README.rst

```diff
@@ -16,15 +16,10 @@ applying deep learning techniques on seismic waveform data.
 
 
 
-.. image:: https://travis-ci.org/yewsg/yews.svg?branch=master
-    :target: https://travis-ci.org/yewsg/yews
```

Comment:
Why do we want to remove the badges here?

Reply:
Resolve. Please check it again. Thanks.